webusb-serial/application.js

```diff
@@ -17,11 +17,17 @@
 
     let currentReceiverLine;
 
-    function appendLine(linesId, text) {
+    function appendLines(linesId, text) {
+      const lines = text.split('\r');
```

Comment:
we should split the lines by '\n' instead. \r is (more) specific to windows than Linux/macOS.